numpy/lib/financial.py

```diff
@@ -45,6 +46,8 @@ def _convert_when(when):
 
 
 def _fv_dispatcher(rate, nper, pmt, pv, when=None):
+    warnings.warn('fv is deprecated and will be removed from NumPy 1.20.',
```

Comment:
I think saying _"use `numpy_financial.pv` instead"_ as part of this message would be worthwhile

Reply:
I agree.  I'll add something like that.

Reply:
I force-pushed an update in which the message is expanded to include information about the recommended alternative.